doc/source/contributing.rst

```diff
@@ -129,6 +129,16 @@ To resume your work, :command:`cd` into your local Git repository and call:
     $ source .tox/3/bin/activate
 
 
+Alternatively, you can launch single commands inside the virtualenv without
+sourcing it directly:
+
+.. code:: shell-session
+
+   $ tox -e devel -- /usr/bin/emacs README.rst
+   $ # or launch kiwi:
+   $ tox -e devel -- kiwi-ng
```

Comment:
I think we should give users not too many "or's" one example how to directly launch from a venv is imho enough, maybe

```
$ tox -e devel -- kiwi-ng --version
```

Thoughts ?

Reply:
Sure, why not.